src/lib/ngsi/Request.cpp

```diff
@@ -75,6 +75,7 @@ const char* requestType(RequestType rt)
   case Ngsi10SubscriptionsConvOp:                   return "Ngsi10SubscriptionsConvOp";
 
   case LogRequest:                                  return "Log";
+  case LogLevelRequest:                             return "LogLevel";
   case VersionRequest:                              return "Version";
```

Comment:
According to https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/admin/management_api.md, the last one is just /log/traces... What is wrong? Documentation or your comment? ;)